hw_03.js

```diff
@@ -0,0 +1,110 @@
+// 1. Напишіть функцію addThemAl
+// Вона буде знаходити суму усіх своїх аргументів незалежно від їх кількості (але без використання вбудованого об'єкту Math).
+// Використайте оператор розширення.
+// 1.1 З використанням циклу.
+
+// console.log(addThemAll(2, 4)); // 6
+// console.log(addThemAll(1, 2, 3, 4)); // 10
+// console.log(addThemAll(5, 5, 10)); // 20
+
+// function addThemAll(...args) {
+//     let sum = 0
+//     for (let arg of args) sum += arg;
```

Comment:
```suggestion
//     for (let arg of args) {
    sum += arg;
}
```
Дуже рекомендую звикати до такого форматування, і виділяти блоки коду явно, на проєктах 99.99% що у вас будуть саме такі налаштування лінтерів

Reply:
Виправила.